ili9341.cpp

```diff
@@ -261,13 +261,22 @@ ili9341_init(void)
 	delay(5);
   }
 }
-
+#if 0
 static inline uint32_t __REV16(uint32_t rev)
 {
 	uint32_t a = __builtin_bswap16(rev >> 16);
 	uint32_t b = __builtin_bswap16(rev & 0xFFFF);
 	return (a << 16) | (b);
 }
+#else

```

Comment:
Just drop my ulgy code and use only use the inline asm. Not need to keep this around.